File: MG/Sandbox/Aleksa Savic/MiniApp/MiniAplikacija/MiniAplikacija/Models/Festival.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace MiniAplikacija.Models
{
    public class Festival
    {
        [Key]
        public int idFest { get; set; }
        public string nazivFestivala { get; set; }
    }
}

File: MG/Sandbox/Aleksa Savic/mini_app_flutter/lib/Models/Festival.dart
class Festival{
  int idFest;
  String nazivFestivala;

  Festival(nazivFestivala)
  {
    this.nazivFestivala=nazivFestivala;
  }

  Festival.idFest(idFest,nazivFestivala)
  {
    this.idFest=idFest;
    this.nazivFestivala=nazivFestivala;
  }

  Map<String, dynamic> toMap(){
    var map=Map<String,dynamic>();
    map['idFest']=idFest;
    map['nazivFestivala']=nazivFestivala;
    return map;
  }

  Festival.fromObject(dynamic o){
    this.idFest=o['idFest'];
    this.nazivFestivala=o['nazivFestivala'];
  }
}

